vmdb/spec/factories/template_openstack.rb

```diff
@@ -1,11 +1,3 @@
 FactoryGirl.define do
-  factory :template_openstack do
-    sequence(:name) { |n| "template_#{seq_padded_for_sorting(n)}" }
-    location        "unknown"
-    uid_ems         { MiqUUID.new_guid }
-    vendor          "openstack"
-    template        true
-    raw_power_state "never"
-    cloud           true
-  end
+  factory(:template_openstack, :class => "TemplateOpenstack", :parent => :template_cloud) { vendor "openstack" }
```

Comment:
Can we not inline the factories?  It's a common pattern to keep it multiline, and this puts it off the end of the page anyway.